agent/command.go

```diff
@@ -139,7 +139,7 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 				}
 			}
 		case <-ctx.Done():
-			tc.logger.Task().Errorf("Command stopped early: %s", ctx.Err())
+			tc.logger.Task().Errorf("Command stopped early, idle timeout duration of %d seconds has been reached: %s", int(tc.timeout.idleTimeoutDuration.Seconds()), ctx.Err())
```

Comment:
Sorry to also add this: I think it's possible to hit this block without hitting timeout (aborting the task might do it, or maybe something else). It may be good to check if ctx.Err() == context.DeadlineExceeded before adding the timeout part of the error.